charts/fulcrum/templates/network-policy.yaml

```diff
@@ -0,0 +1,16 @@
+kind: NetworkPolicy
```

Comment:
Did fulcrum not work without the network policy? Just confirming that it works

Reply:
removed in https://github.com/GaloyMoney/charts/pull/2418/commits/65ece7ebcbb5d91336b280cada54b8101d17dfab